sensehat/src/main/java/com/google/android/things/contrib/driver/sensehat/SenseHat.java

```diff
@@ -23,12 +23,17 @@
  */
 @SuppressWarnings({"unused", "WeakerAccess"})
 public class SenseHat {
-    public static final int I2C_ADDRESS =  0x46;
-    public static final String BUS_DISPLAY = "I2C1";
+    public static final int I2C_DISPLAY_ADDRESS =  0x46;
+    public static final int I2C_LPS25H_ADDRESS =  0x46;
+    public static final String BUS_NAME = "I2C1";
```

Comment:
We should make this bus name a parameter that can be provided, for a variety of devices with different bus names.

Reply:
Yes we should! However, this is from the original code and I daresay a common issue through the other libraries as well. Maybe there should be a global device data object to address this?
Meanwhile I've provided overloaded factory methods.